pypeit/scripts/ql_keck_deimos.py

```diff
@@ -0,0 +1,233 @@
+#!/usr/bin/env python
+#
+# See top-level LICENSE file for Copyright information
+#
+# -*- coding: utf-8 -*-
+"""
+This script runs PypeIt *fast* on a set of DEIMOS images
+"""
+import os
+import glob
+import copy
+import yaml
+import io
+import numpy as np
+import argparse
+
+from pypeit.scripts import scriptbase
+
+from pypeit.scripts.utils import Utilities
+from pypeit.par import PypeItPar
+from pypeit.par.util import parse_pypeit_file
+from pypeit.pypeitsetup import PypeItSetup
+from pypeit import msgs
+from pypeit.scripts import run_pypeit
+from pypeit.par.util import make_pypeit_file
+from pypeit import utils
+
+from IPython import embed
+
+class QLKeckDEIMOS(scriptbase.ScriptBase):
+
+    @classmethod
+    def name(cls):
+        """
+        Return the name of the executable.
+        """
+        return 'pypeit_ql_keck_deimos'
+
+    @classmethod
+    def get_parser(cls, width=None):
+        import argparse
+        parser = super().get_parser(description='Pypeit QL for Keck/DEIMOS',
+                                    width=width, formatter=argparse.RawDescriptionHelpFormatter)
+        parser = argparse.ArgumentParser(description='Script to run PypeIt in QuickLook on a set of '
+                                                    'Keck/DEIMOS files',
+                                        formatter_class=argparse.ArgumentDefaultsHelpFormatter)
+        parser.add_argument('full_rawpath', type=str, help='Full path to the raw files')
```

Comment:
I'm not sure I understand why we have here both `full_rawpath` and `root`. It seems it would be easier to just have `root` (or `full_rawpath`) like it's done in `pypeit_setup`.

Reply:
The reason (which is not a great one) is that we do may call `pypeit_setup` which should have `root` but we also call `run_pypeit` on a science file which does not..

Reply:
I see no issue with having both, although changing the help to clarify `Full path to the directory containing the raw files` might make it clearer to users how to use both

Reply:
clarification added